Tests/AWSSDKSwiftCoreTests/MetaDataServiceTests.swift

```diff
@@ -7,17 +7,32 @@
 
 #if os(Linux)
 import XCTest
+import NIO
 @testable import AWSSDKSwiftCore
 
 class MetaDataServiceTests: XCTestCase {
-  static var allTests : [(String, (MetaDataServiceTests) -> () throws -> Void)] {
-      return [
-        ("testMetaDataServiceForECSCredentials", testMetaDataServiceForECSCredentials),
-        ("testMetaDataServiceForInstanceProfileCredentials", testMetaDataServiceForInstanceProfileCredentials)
-      ]
-  }
+    static var allTests : [(String, (MetaDataServiceTests) -> () throws -> Void)] {
+        return [
+            ("testMetaDataServiceForECSCredentials", testMetaDataServiceForECSCredentials),
+            ("testMetaDataServiceForInstanceProfileCredentials", testMetaDataServiceForInstanceProfileCredentials),
+            ("testMetaDataGetCredential", testMetaDataGetCredential)
+        ]
+    }
 
-  func testMetaDataServiceForECSCredentials() {
+    // Disabling cannot guarantee that 169.254.169.254 is not a valid IP on another network
+    func testMetaDataGetCredential() {
+        if ProcessInfo.processInfo.environment["TEST_EC2_METADATA"] != nil {
+            do {
+                let group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+                _ = try MetaDataService.getCredential(eventLoopGroup: group).wait()
+            } catch {
+                print(error)
+                XCTFail()
```

Comment:
instead of print, use 
```
XCTFail("Unexpected error: \(error)")
```

This way the error is shown directly in the xcode red bubble next to the failing error. No search in console required.

Reply:
This test nevers run in XCode. It is only ever of use on an EC2 instance. I'll change it anyway